packages/fuels/tests/test_projects/append_message_output/src/main.sw

```diff
@@ -0,0 +1,14 @@
+contract;
```

Comment:
I could not find where we use this `contract`. Am I missing something ?

Reply:
No you're not missing anything, I'm going to remove it. e don't need to test Sway opcodes in the SDK (duh), I just got confused haha.